requirements.yml

```diff
@@ -3,3 +3,4 @@ collections:
   - name: middleware_automation.redhat_csp_download
     version: ">=1.2.1"
   - name: community.general
+  - name: community.docker
```

Comment:
Nop, this requirement is not for molecule to run, but the dependency of the collection itself. The wild fly collection does NOT require the docker collection to run.

Reply:
Done, Thanks